<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="TRVLHGN9NN2" name="ngtutorial.pckge.AssertTestngPart1" tests="2" failures="1" timestamp="2021-08-06T07:38:55 IST" time="0.027" errors="0">
  <testcase name="verifyTitle" time="0.005" classname="ngtutorial.pckge.AssertTestngPart1"/>
  <system-out/>
  <testcase name="softASsert1" time="0.022" classname="ngtutorial.pckge.AssertTestngPart1">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Electronics, Cars, Fashio, Collectibles &amp; More | eBay] but found [Electronics, Cars, Fashion, Collectibles &amp; More | eBay],
	Url failed expected [https://www.eay.com/] but found [https://www.ebay.com/]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Electronics, Cars, Fashio, Collectibles & More | eBay] but found [Electronics, Cars, Fashion, Collectibles & More | eBay],
Url failed expected [https://www.eay.com/] but found [https://www.ebay.com/]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at ngtutorial.pckge.AssertTestngPart1.softASsert1(AssertTestngPart1.java:58)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- softASsert1 -->
  <system-out/>
</testsuite> <!-- ngtutorial.pckge.AssertTestngPart1 -->
